srml/council/src/voting.rs

```diff
@@ -129,32 +158,36 @@ decl_event!(
 	pub enum Event<T> where <T as system::Trait>::Hash {
 		/// A voting tally has happened for a referendum cancellation vote.
 		/// Last three are yes, no, abstain counts.
-		TallyCancelation(Hash, u32, u32, u32),
+		TallyCancellation(Hash, u32, u32, u32),
 		/// A voting tally has happened for a referendum vote.
 		/// Last three are yes, no, abstain counts.
 		TallyReferendum(Hash, u32, u32, u32),
 	}
 );
 
 impl<T: Trait> Module<T> {
+	/// Returns true if there are one or more vetos in effect on a proposal.
 	pub fn is_vetoed<B: Borrow<T::Hash>>(proposal: B) -> bool {
 		Self::veto_of(proposal.borrow())
 			.map(|(expiry, _): (T::BlockNumber, Vec<T::AccountId>)| <system::Module<T>>::block_number() < expiry)
 			.unwrap_or(false)
 	}
 
+	/// Returns true if `who` will still be a councillor at block `n`.
 	pub fn will_still_be_councillor_at(who: &T::AccountId, n: T::BlockNumber) -> bool {
 		<Council<T>>::active_council().iter()
 			.find(|&&(ref a, _)| a == who)
 			.map(|&(_, expires)| expires > n)
 			.unwrap_or(false)
 	}
 
+	/// Returns true if `who` is a councillor.
 	pub fn is_councillor(who: &T::AccountId) -> bool {
 		<Council<T>>::active_council().iter()
 			.any(|&(ref a, _)| a == who)
 	}
 
+	/// Tally the votes for a proposal. Tuple returned is yay, nay, and abstain votes, respectively.
 	pub fn tally(proposal_hash: &T::Hash) -> (u32, u32, u32) {
```

Comment:
as per previous comment, consider changing to `pub fn tally(proposal_hash: &T::Hash) -> (YaysCount, NaysCount, AbstainsCount) {`, and then remove the comment `Tuple returned is yay, nay, and abstain votes, respectively.` since the we now have self-describing code.